endpoints/getting-started/app.yaml

```diff
@@ -1,8 +1,6 @@
 runtime: go
 env: flex
 
-beta_settings:
-  # Enable Google Cloud Endpoints API management.
-  use_endpoints_api_management: true
-  # Specify the Open API specification.
-  endpoints_swagger_spec_file: openapi.yaml
+endpoints_api_service:
+  name: ENDPOINTS SERVICE-NAME # To be replaced.
+  config_id: ENDPOINTS CONFIG-ID # To be replaced.
```

Comment:
The commend could probably be more descriptive. The intention is have the users go through the Quickstart, run "gcloud service-management deploy openapi.yaml" first, and then take the output of that command (service name and config_id) and update app.yaml 

I left the space (ENDPOINTS" "SERVICE-NAME) in there intentionally because I want the "gcloud app deploy" command to fail if the users (who should be going through the Quickstart) do not update their app.yaml for whatever reason. The command would fail due to the AppInfo regex validation.

We are in the process of adding a better client side validation to make sure the name and config_id provided are actually valid, but unfortunately adding such a validation causes other concerns, so we are still working on it. In the meanwhile, if you provide an invalid (but syntactically correct) config_id, the deployment would proceed and fail much later after spinning for 20 minutes... which is horrible.

Let me know if you'd like to discuss the details.